index.html

```diff
@@ -20,28 +22,47 @@
             <div>
                 <ul class="bars">
                     <li>
-                        <a href="#"><i class="fa fa-bars fa-2x" aria-hidden="true"></i></a>
+                        <button class="open"><i class="fa fa-bars fa-3x" aria-hidden="true"></i></button>
+                    </li>
+                    <li class="close-page">
+                        <div>
+                            <div class="close">
+                                <button class="close"><i class="fa fa-close fa-2x"></i></button>
+                            </div>
+                            <div class="menu-list">
+                                <ul>
+                                    <li class="menu-link"><a href="#portfolio">Portfolio</a></li>
+                                    <li><a href="#about" class="menu-link">About</a></li>
+                                    <li><a href="#contact" class="menu-link">Contact</a></li>
+                                    <li class="menu-divider">
+                                        <div></div>
+                                    </li>
+                                </ul>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_5](https://user-images.githubusercontent.com/69638013/161509772-8d06045b-1c2e-429a-a19f-5a13841832a9.png) |  ![port](https://user-images.githubusercontent.com/69638013/161744374-32538fea-0b68-4f92-a00f-982fb2f8d5c8.png)


- [ ] Kindly make the background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿 
- [ ] It would be better for the background not to be scrollable when the mobile nav is open 👍🏿 
- [ ] Kindly remove the menu divider at the end of the page as it is not part of the design

- Adding these functionalities will make your design good and look exactly like the Figma design, great job so far 💪🏿

Reply:
@prolajumokeoni All has been incorporated, thanks.